config/environment.js

```diff
@@ -8,9 +8,14 @@ module.exports = function (environment) {
     rootURL: '/',
     locationType: 'auto',
     moment: {
-      includeLocales: ['nl'],
+      includeLocales: ['nl-be'],
```

Comment:
Hebben we zowel `moment` als `ember-moment` in config nodig? Ik denk dat er eentje overbodig is? (zie ook https://github.com/adopted-ember-addons/ember-moment/issues/375)

Reply:
Nog even getest en ziet er wel goed uit met de dubbele config eruit.